landlab/components/depth_dependent_diffusion/hillslope_depth_dependent_linear_flux.py

```diff
@@ -19,7 +19,7 @@ class DepthDependentDiffuser(Component):
 
     .. math::
 
-        q_s = D S H^* (1.0 - exp( H / H^*)
+        q_s = - D S H^* (1.0 - exp( - H / H^*)
```

Comment:
Not sure about the leading minus. If S = dz/dx, then it's needed, but if S is positive downward, i.e. S = -dz/dx, then I think it should not be there.

Reply:
In the actual code we calculate as:

```python
        # Calculate flux
        self._flux[:] = (
            -self._K
            * slope
            * self._soil_transport_decay_depth
            * (1.0 - np.exp(-H_link / self._soil_transport_decay_depth))
        )
```

implying S = dzdx. Probably worth explicitly stating this to avoid confusion. Will add.